README.md

```diff
@@ -5,17 +5,26 @@ Logo files for [Jekyll](http://jekyllrb.com).
 
 ![](https://github.com/cobyism/jekyll-logo/blob/master/jekyll-logo-dark-solid.png?raw=true)
 ![](https://github.com/cobyism/jekyll-logo/blob/master/jekyll-logo-light-transparent.png?raw=true)
```

Comment:
Derp. Just realised this still points to my old version of this repo. We should probably update that. :sweat_smile: